tests/integration_tests/performance/test_network_latency.py

```diff
@@ -236,3 +236,4 @@ def _g2h_send_ping(context):
         handle_failure(file_dumper, err)
 
     file_dumper.dump(result)
+    basevm.kill()
```

Comment:
`handle_failure` can raise an exception (see [here](https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/performance/utils.py)), so I think this should go into a `finally` block

Reply:
Good catch. Will fix.